apps/site/lib/site_web/channels/vehicle_channel.ex

```diff
@@ -41,6 +41,7 @@ defmodule SiteWeb.VehicleChannel do
     route = Routes.Repo.get(vehicle.route_id)
     stop_name = get_stop_name(vehicle.stop_id)
     trip = Schedules.Repo.trip(vehicle.trip_id)
+    prediction = List.first(Predictions.Repo.all(trip: vehicle.trip_id))
```

Comment:
I don't think this is right -- I think you need to at least pass the stop id as well.

Reply:
This worked for CR w/o the stop id. Since these are live updates, it feels like it should always work for a given trip, but maybe better to be defensive.

I did check this out on Red line pages without the stop and it looks like it works correctly as well, but I discovered a different issue with language that I'm going to bring up to product anyway.